pkg/operation/botanist/component/metricsserver/metrics_server.go

```diff
@@ -262,6 +266,9 @@ func (m *metricsServer) computeResourcesData() (map[string][]byte, error) {
 					managedresources.LabelKeyOrigin: managedresources.LabelValueGardener,
 					v1beta1constants.GardenRole:     v1beta1constants.GardenRoleSystemComponent,
 				}),
+				Annotations: map[string]string{
+					"gardener-resource-manager.gardener.cloud/preserveResources": "true",
```

Comment:
Can you use an exported constant from the `github.com/gardener/gardener-resource-manager` repository instead?